EIPS/eip-747.md

```diff
@@ -0,0 +1,132 @@
+---
+eip: 747
+title: Add wallet_watchAsset to Provider
+author: Dan Finlay (@danfinlay), Esteban Mino (@estebanmino)
+discussions-to: https://ethereum-magicians.org/t/eip-747-eth-watchtoken/1048
+status: Draft
+type: Interface
```

Comment:
Dunno why the validator hasn't caught this, but it should be: `type: Standards Track`

Reply:
Corrected.